Flake8 issues

`flake8==3.8.1` catches a few new flake8 errors that are making CI tests fail:

```python
.\src\pynwb\legacy\__init__.py:41:1: E402 module level import not at top of file
.\tests\coloredtestrunner.py:94:26: E741 ambiguous variable name 'l'
```

--- a/src/pynwb/legacy/__init__.py
+++ b/src/pynwb/legacy/__init__.py
@@ -38,4 +38,4 @@ def register_map(**kwargs):
         _dec(mapper_cls)
 
 
-from . import io  # noqa: F401
+from . import io  # noqa: F401,E402

--- a/tests/coloredtestrunner.py
+++ b/tests/coloredtestrunner.py
@@ -91,7 +91,7 @@ class Table(object):
         return len(re.sub(r"\033\[[0-9];[0-9];[0-9]{1,2}m", "", x))
 
     def addRow(self, row):
-        rows = [[''] for l in range(len(row))]
+        rows = [[''] for i in range(len(row))]
         maxrows = 0
         for i, x in enumerate(row):
             for j, y in enumerate(x.split("\n")):

